buildscripts/conda_recipes/numba-scipy/meta.yaml

```diff
@@ -30,7 +30,7 @@ test:
     - scipy<1.11.0
     - pytest
     - setuptools
-    - faulthandler             # [py27 and (not (armv6l or armv7l))]
+    - faulthandler # [py27 and (not (armv6l or armv7l))]
```

Comment:
It seems like this should not be needed anymore (as we're not doing anything on Python 2).